assets/css/style.min.css

```diff
@@ -1 +1 @@
\ No newline at end of file
+#minor-publishing-actions,.misc-pub-section.misc-pub-post-status,.misc-pub-section.misc-pub-visibility{display:none}#submitdiv{border:none;display:block;margin-bottom:20px;background:0 0!important}#submitdiv .hndle{background:#fff;border:1px solid #e5e5e5}#minor-publishing{background:#fff;border:1px solid #e5e5e5;border-top:0}#major-publishing-actions{padding:0;border:none;margin-top:20px;background:#fff}#major-publishing-actions .hndle{border:1px solid #e5e5e5}#publishing-actions{padding:8px 12px;border:1px solid #e5e5e5;border-top:0}#timestamp hide,.timestamp-wrap hide{display:none}.button-secondary.dashicons{width:50px}.edw-loading{width:20px;display:inline-block;vertical-align:middle}.postbox[id^=pi-sections-template]{display:none}#pi-section-add-info{display:none}.pi-error-msg{display:none}#pi-section-name{width:232px}#pi-section-add-confirm,#pi-section-name{display:inline-block;vertical-align:middle}.postbox span[contenteditable]{cursor:text;display:inline-block;vertical-align:middle;margin-right:10px}.postbox span[contenteditable].locked{cursor:wait;color:grey}.pi-article-add-info{display:none}.pi-article-results{display:block;clear:both;display:none}.pi-article-title{width:182px}.pi-article-search,.pi-article-title{display:inline-block;vertical-align:middle}select.pi-article-dropdown{max-width:300px}.pi-articles tfoot{display:none}.pi-article-remove,.pi-article-view{visibility:hidden;margin-left:10px}.pi-articles .column-title:hover .pi-article-remove,.pi-articles .column-title:hover .pi-article-view{visibility:visible}.pi-article-remove{color:#a00}.pi-article-remove:hover{color:red}.column-bulk-article-status input.cb-select-all-article-status{margin-left:0}th#bulk-article-status{width:2.2em}th.column-article-status{width:15%}th.column-date{width:auto!important}th.column-title{width:28%}.article-export-buttons{clear:left;margin-top:15px}
```

Comment:
I wanted to manually review what's changed here, I think it's fine (although a little odd IMO) but just wanted to note things in case it becomes a problem later:

1. Rather than grouping a bunch of selectors for the same usage of a given property (border, display), the new build seems to prefer to keep all properties for a selector together.
2. In `#publishing-actions`, it goes from `padding: 8px 9pt` to `padding: 8px 12px`. The Sass file has `8px 12px` so that's a fix IMO.
3. `.pi-article-results` now keeps both `display: block` and `display: none` as in the source - I actually cannot figure out where `pi-article-results` is used or generated at the moment, so that probably deserves investigation at some point. It may be that it's a default-inline element being made a block element and hidden/shown using `display` so it kind of makes sense, but it's also weird and would have been broken before. We may not need this ruleset at all.